templates/index.html

```diff
@@ -112,7 +112,7 @@ <h3>Stream games to millions</h3>
       <h3>Virtualise phones and tablets</h3>
       <ul class="p-list">
         <li class="p-list__item is-ticked">Run virtual Android instances in the cloud</li>
-        <li class="p-list__item is-ticked">Client interface available as native or web application</li>
+        <li class="p-list__item is-ticked">Client interface available as built-in or web application</li>
```

Comment:
I think "native app" is used by the industry,. For example, Android Studio even uses the word as a part of a tool: https://developer.android.com/ndk/guides

Reply:
Yes, you are right. I have now removed the check for 'native' from the action.